.circleci/config.yml

```diff
@@ -379,7 +379,69 @@ jobs:
           name: 'Save Yarn Package Cache'
           paths:
             - ~/.cache/yarn
-
+  cloud-lighthouse:
+    machine:
+      image: ubuntu-2004:202008-01
+      docker_layer_caching: true
+    parallelism: 10
```

Comment:
do we need to run this on 10 machines or will one run do?

Reply:
Will remove this. One run is good enough.